### PR TITLE
HBASE-22003 Fix flaky test TestVerifyReplication.testHBase14905

### PR DESCRIPTION
**Pull request by infraio**

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 25 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 256 | master passed |
| +1 | compile | 27 | master passed |
| +1 | checkstyle | 19 | master passed |
| +1 | shadedjars | 271 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 40 | master passed |
| +1 | javadoc | 18 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 237 | the patch passed |
| +1 | compile | 25 | the patch passed |
| +1 | javac | 25 | the patch passed |
| +1 | checkstyle | 19 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 267 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 975 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.0.3 3.1.2. |
| +1 | findbugs | 47 | the patch passed |
| +1 | javadoc | 18 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 929 | hbase-mapreduce in the patch failed. |
| +1 | asflicense | 17 | The patch does not generate ASF License warnings. |
| | | 3507 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.snapshot.TestExportSnapshotNoCluster |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-259/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/259 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux 683b62051628 4.4.0-138-generic #164-Ubuntu SMP Tue Oct 2 17:16:02 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 1c1638f698 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-259/1/artifact/out/patch-unit-hbase-mapreduce.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-259/1/testReport/ |
| Max. process+thread count | 5535 (vs. ulimit of 10000) |
| modules | C: hbase-mapreduce U: hbase-mapreduce |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-259/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by infraio** [comment]
The failed ut not related.

File: hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/replication/TestVerifyReplication.java [hunk 1]
```diff
@@ -252,11 +252,12 @@ public void testHBase14905() throws Exception {
     // normal Batch tests
     byte[] qualifierName = Bytes.toBytes("f1");
     Put put = new Put(Bytes.toBytes("r1"));
-    put.addColumn(famName, qualifierName, Bytes.toBytes("v1002"));
+    long ts = System.currentTimeMillis();
+    put.addColumn(famName, qualifierName, ts+1, Bytes.toBytes("v1002"));
```
Comment (Apache9) [inline hbase-mapreduce/src/test/java/org/apache/hadoop/hbase/replication/TestVerifyReplication.java]: Formatting issue?`ts+1` ->`ts + 1`

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 106 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | hbaseanti | 0 |  Patch does not have any anti-patterns. |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 1 new or modified test files. |
||| _ master Compile Tests _ |
| +1 | mvninstall | 334 | master passed |
| +1 | compile | 41 | master passed |
| +1 | checkstyle | 31 | master passed |
| +1 | shadedjars | 351 | branch has no errors when building our shaded downstream artifacts. |
| +1 | findbugs | 66 | master passed |
| +1 | javadoc | 24 | master passed |
||| _ Patch Compile Tests _ |
| +1 | mvninstall | 355 | the patch passed |
| +1 | compile | 45 | the patch passed |
| +1 | javac | 45 | the patch passed |
| +1 | checkstyle | 29 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedjars | 418 | patch has no errors when building our shaded downstream artifacts. |
| +1 | hadoopcheck | 1451 | Patch does not cause any errors with Hadoop 2.8.5 2.9.2 or 3.0.3 3.1.2. |
| +1 | findbugs | 72 | the patch passed |
| +1 | javadoc | 23 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 2849 | hbase-mapreduce in the patch failed. |
| +1 | asflicense | 20 | The patch does not generate ASF License warnings. |
| | | 6612 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hbase.snapshot.TestMobSecureExportSnapshot |
|   | hadoop.hbase.snapshot.TestExportSnapshotWithTemporaryDirectory |
|   | hadoop.hbase.snapshot.TestSecureExportSnapshot |
|   | hadoop.hbase.snapshot.TestMobExportSnapshot |
|   | hadoop.hbase.snapshot.TestExportSnapshot |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-259/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/259 |
| Optional Tests |  dupname  asflicense  javac  javadoc  unit  findbugs  shadedjars  hadoopcheck  hbaseanti  checkstyle  compile  |
| uname | Linux 46e43dae132e 4.4.0-131-generic #157~14.04.1-Ubuntu SMP Fri Jul 13 08:53:17 UTC 2018 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | /testptch/patchprocess/precommit/personality/provided.sh |
| git revision | master / 1c1638f698 |
| maven | version: Apache Maven 3.5.4 (1edded0938998edf8bf061f1ceb3cfdeccf443fe; 2018-06-17T18:33:14Z) |
| Default Java | 1.8.0_181 |
| findbugs | v3.1.11 |
| unit | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-259/2/artifact/out/patch-unit-hbase-mapreduce.txt |
|  Test Results | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-259/2/testReport/ |
| Max. process+thread count | 4360 (vs. ulimit of 10000) |
| modules | C: hbase-mapreduce U: hbase-mapreduce |
| Console output | https://builds.apache.org/job/HBase-PreCommit-GitHub-PR/job/PR-259/2/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**